File: CPP/LH05/ReverseNum.cpp
#include <iostream>
using namespace std;
int main(){
    int Num;
    cout<<"Enter Your Number : ";
    cin>>Num;
    cout<<"Reverse Numeber : ";
    int sum = 0; 

    while(Num){
        int rem = Num%10;
        sum = sum*10+rem;
        Num /= 10; 
    }
    cout<<sum<<endl;
}

File: Java/ApnaJava/Loop/ReverseNum.java
package Loop;
import java.util.*;

public class ReverseNum {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter Your Number : ");
        int num = sc.nextInt();
        int reverse = 0; 

        while(0<num){
            int extract = num % 10 ;
            reverse  = (reverse*10)+extract;
            num /= 10;
        }

        System.out.println("Your Reverse Number : " + reverse);
    }
}
